Swift Code Challenge/Views/UsersListView.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  UsersListView.swift
+//  Swift Code Challenge
+//
+//  Created by Luiz Eduardo Mello dos Reis on 29/06/22.
+//
+
+import SwiftUI
+
+struct UsersListView: View {
+    @ObservedObject var viewModel: UsersListViewModelAsync = UsersListViewModelAsync()
+    @State private var showDevice = false
+    @State private var selection: Person? = nil
+    
+    
+    var body: some View {
+        ZStack{
+            NavigationView{
+                if(viewModel.personList.isEmpty){
```

Comment:
No need to put in parentheses. I know other languages, such as Objective-C, require you to use parentheses more but they're not needed in Swift.

```suggestion
                if viewModel.personList.isEmpty {
```